apps/nowcasting-app/pages/api/auth/[...auth0].ts

```diff
@@ -30,7 +30,8 @@ export default withSentry(
 
         await handleLogin(req, res, {
           authorizationParams: {
-            redirect_uri: redirectUri
+            redirect_uri: redirectUri,
+            audience: "https://nowcasting-api-eu-auth0.com/"
```

Comment:
is this the same for dev and pro?

Reply:
It won't be I don't think - that was that note to self/team earlier that this needs to live in an Env Var on Vercel before we merge to Production